packages/template-tag-codemod/src/index.ts

```diff
@@ -105,7 +110,11 @@ export async function ensureAppSetup() {
     process.exit(-1);
   }
   if (!pkg.packageJSON.exports) {
-    throw new Error(`must use package.json exports for self-resolvability. Plase add this to package.json:
+    pkg.packageJSON.exports = {
```

Comment:
I guess we won't need this once https://github.com/embroider-build/embroider/pull/2279 is merged 👍